singer-io/tap-stripe
# TDL-17879 added authentication before discovery mode and unittests

# Description of change
- Added credentials in discovery mode.
- Throws an error if the creds are not correct.
- Added unittests for the same

# Manual QA steps
 - Check that with wrong credentials the discovery mode does not create a catalog file.
 - Check that with correct credentials the discovery mode creates correct catalog file.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
